bHY6yh-d28515/cod-yyyiou_bHY6yh-d28515.xml

```diff
@@ -0,0 +1,313 @@
+            <respStmt>
+               <name ref="#jeffreycwitt">Jeffrey C. Witt</name>
+               <resp>Transcription Editor</resp>
+               <resp>TEI Encoder</resp>
+            </respStmt>
+            <respStmt>
+              <name ref="#alasdairwsliter">Alasdair W. Sliter</name>
+              <resp>Transcription Editor</resp>
+              <resp>TEI Encoder</resp>
+            </respStmt>
+         </titleStmt>
+         <editionStmt>
+            <edition n="0.0.0-dev">
+               <title>Rubrica 4</title>
+               <date when="2023-08-05">August 5, 2023</date>
+            </edition>
+         </editionStmt>
+         <publicationStmt>
+            <authority>SCTA</authority>
+            <availability status="free">
+               <p>Published under a <ref target="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</ref>
+               </p>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <listWit>
+               <witness xml:id="C" n="cod-yyyiou">Vatican, Biblioteca Apostolica Vaticana, Ross. 429 (IX.119)</witness>
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2023-08-05" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#C" n="38-r"/>
+            <cb ed="#C" n="a"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-d28515"><!-- p2l3-r4 -->
+            <head xml:id="bHY6yh-d28515-Hd1e108">Rubrica 4</head>
+            <head xml:id="bHY6yh-d28515-Hd1e111" type="question-title">De parentibus et liberis</head>
+            <p xml:id="bHY6yh-d28515-d1e114">
+              De dilectionem et hono<lb ed="#C" break="no"/>re 
+              parentum et cognatorum
+            </p>
+            <!--next paragraph begins on the same line as the previous paragraph-->
+            <p xml:id="bHY6yh-d28515-d1e119">
+              <ref xml:id="bHY6yh-d28515-Rd1e115">IN exo<lb ed="#C" break="no"/>do 
+              capitulo 20</ref> <quote xml:id="bHY6yh-d28515-Qd1e123">
+              <lb ed="#C"/>honora patrem tuum et matrem tuam ut sis
+              <lb ed="#C"/>longeuus super terram quam dominus deus tuus
```

Comment:
If the quote doesn't start until the next line, it should go after the line break, like so...

```suggestion
              capitulo 20</ref> 
              <lb ed="#C"/><quote xml:id="bHY6yh-d28515-Qd1e123">honora patrem tuum et matrem tuam ut sis
              <lb ed="#C"/>longeuus super terram quam dominus deus tuus
```